src/sql/src/rbac.rs

```diff
@@ -909,6 +909,27 @@ fn generate_rbac_requirements(
             ownership: vec![ObjectId::Item(*id_a), ObjectId::Item(*id_b)],
             ..Default::default()
         },
+        Plan::AlterSchemaRename(plan::AlterSchemaRenamePlan {
+            cur_schema_spec,
+            new_schema_name: _,
+        }) => {
+            let privileges = match cur_schema_spec.0 {
+                ResolvedDatabaseSpecifier::Id(db_id) => vec![(
+                    SystemObjectId::Object(ObjectId::Database(db_id)),
+                    AclMode::CREATE,
+                    role_id,
+                )],
+                ResolvedDatabaseSpecifier::Ambient => {
+                    vec![(SystemObjectId::System, AclMode::CREATE, role_id)]
```

Comment:
This isn't quite correct. This is reachable for temporary schemas right? If so, we should not require any additional privileges here.

Reply:
Sweet, updated!